specification/sql/resource-manager/Microsoft.Sql/preview/2015-05-01-preview/examples/VirtualClusterGet.json

```diff
@@ -0,0 +1,26 @@
+{
+    "parameters": {
+        "subscriptionId": "20d7082a-0fc7-4468-82bd-542694d5042b",
+        "resourceGroupName": "testrg",
+        "virtualClusterName": "vc-subnet1-f769ed71-b3ad-491a-a9d5-26eeceaa6be2",
+        "api-version": "2015-05-01-preview"
+    },
+    "responses": {
+        "200": {
+            "body": {
+                "properties": {
+                    "subnetId": "/subscriptions/20D7082A-0FC7-4468-82BD-542694D5042B/resourceGroups/testrg/providers/Microsoft.Network/virtualNetworks/vnet1/subnets/subnet1",
+                    "family": "Gen4",
+                    "childResources": [
+                        "/subscriptions/20D7082A-0FC7-4468-82BD-542694D5042B/resourceGroups/testrg/providers/Microsoft.Sql/managedInstances/testinstance1",
+                        "/subscriptions/20D7082A-0FC7-4468-82BD-542694D5042B/resourceGroups/testrg/providers/Microsoft.Sql/managedInstances/testinstance2"
+                    ]
+                },
+                "location": "onebox",
```

Comment:
It would be preferable to use a real location here, e.g. "westeurope"

Reply:
Done.